packages/plugin-workflow-approval/src/client/common/constants/approval-todo-status-options.ts

```
@@ -0,0 +1,41 @@
+    value: APPROVAL_TODO_STATUS.REJECTED,
+    label: tval('Rejected'),
+    color: 'red',
+  },
+  {
+    value: APPROVAL_TODO_STATUS.WITHDRAWN,
+    label: tval('Withdrawn'),
+  },
+];
+export const approvalTodoStatusMap = approvalTodoStatusOptions.reduce(
+  (configMap, option) => ({
+    ...configMap,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)